src/analyzer.ts

```diff
@@ -165,7 +145,7 @@ export class StreamAnalyzer extends Transform {
   }
 
   isFragment(file: File): boolean {
```

Comment:
this makes the "fragments" vs. "allFragments" collision even more confusing (https://github.com/Polymer/polymer-project-config/pull/1#r79076957)